curriculum/challenges/english/25-front-end-development/quiz-tooling-and-deployment/66fd6f0370213be90fed82ec.md

```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is a bundler?
 
 #### --distractors--
 
-Placeholder distractor 1
+A development tool that bundles text for a website.
 
 ---
 
-Placeholder distractor 2
+A development tool that is used to generate bundles of images for a website
 
 ---
 
-Placeholder distractor 3
+A development tool that writes bundles of javascript code for you
 
 #### --answer--
 
-Placeholder answer
+A development tool that combines many JavaScript code files into a single one
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What type of graph does a bundler generate?
 
 #### --distractors--
 
-Placeholder distractor 1
+An unweighted graph
 
 ---
 
-Placeholder distractor 2
+A weighted graph
 
 ---
 
-Placeholder distractor 3
+An undependency graph
 
 #### --answer--
 
-Placeholder answer
+A dependency graph
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of these is not an example of perfomance optimization feature of a bundler?
 
 #### --distractors--
 
-Placeholder distractor 1
+Code splitting
 
 ---
 
-Placeholder distractor 2
+Hot module replacement
 
 ---
 
-Placeholder distractor 3
+Minification
 
 #### --answer--
 
-Placeholder answer
+Code generation
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The process of generating a relationship map of all the served files in a project is called?
 
 #### --distractors--
 
-Placeholder distractor 1
+Mappification
 
 ---
 
-Placeholder distractor 2
+Relationship resolution
 
 ---
 
-Placeholder distractor 3
+Data resolution
 
 #### --answer--
 
-Placeholder answer
+Dependency resolution
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which file does a bundler use to start traversing your code and understand its dependencies?
 
 #### --distractors--
 
-Placeholder distractor 1
+A CSS file
 
 ---
 
-Placeholder distractor 2
+An `index.html` file
 
 ---
 
-Placeholder distractor 3
+Any JavaScript file
 
 #### --answer--
 
-Placeholder answer
+An Entry File
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+GitHub Actions offers features that let you _______
```

Comment:
Not sure about this one. Technically all of the answers, except maybe user access control, would fit.